### PR TITLE
Update project name, copyright

### LINKED ISSUE #54: [Other] Allow access to webhooks for readthedocs
I recently set up the [readthedocs site](https://paperless-ngx.readthedocs.io/en/latest/). Unfortunately, I don't have access to the webhooks settings of the project. 
I requested access to it from the project owner. When granted, the docs will automatically update.
Also we should change some items in https://github.com/paperless-ngx/paperless-ngx/blob/master/docs/conf.py 
(namely `project = u'Paperless-ng'` and `copyright = u'2021, Daniel Quinn, Jonas Winkler'`)

Should we just add `paperless-ngx team` to it?

### PR DESCRIPTION
**Pull request by qcasey**
Closes #54 

Should these be merged into `dev`?

### CODE REVIEW

**Comment by gador** [comment]
The docs are build with `master` branch set, so the changes wont be reflected until we merge `dev` into `master`. 

I'd suggest merging them into `dev` right now and collect any other changes we want to include in the docs and merge them later all together into `master`. What do you think?

**Comment by qcasey** [comment]
Yep fine by me

**Comment by dblitt** [comment]
Should we change "2021" to "2015-2022"? Obviously 2022 is the current year, and it looks like Daniel Quinn's contributions go back to December 2015 as per the git history.

**Comment by qcasey** [comment]
Yup definitely, I missed that

**Comment by dblitt** [comment]
I just changed a bunch of references to paperless-ng to paperless-ngx in README.md. Things I noticed that still need to be changed:

- Crowdin button is to the old project
- Same for docker hub
- I had issues pointing the coveralls button to the master branch, so its still the old project
- We seem to be holding on to the paperless-ng section before introducing paperless-ngx, so the links to the docs there still point to paperless-ng
- Crowdin issue (212 on paperless-ng)

**Comment by qcasey** [comment]
Thanks! I added that issue of crowdin to #1. Perhaps we should do the README and coveralls/docker button update in a new PR once those are ready.

I think we're starting with ghcr anyway (https://github.com/paperless-ngx/paperless-ngx/tree/feature-docker-build-fix)

**Review by jovandeginste — COMMENTED** [review]
My remarks

File: README.md [hunk 1]
```diff
@@ -64,29 +73,29 @@ If you want to see paperless-ng in action, [more screenshots are available in th
...
-Read the [documentation](https://paperless-ng.readthedocs.io/en/latest/setup.html#installation) on how to get started.
+Read the [documentation](https://paperless-ngx.readthedocs.io/en/latest/setup.html#installation) on how to get started.
 
 Alternatively, you can install the dependencies and setup apache and a database server yourself. The documenation has a step by step guide on how to do it. Consider giving the Ansible role a shot, this essentially automates the entire bare metal installation process.
 
-# Migrating from Paperless to Paperless-ng
+# Migrating from Paperless to Paperless-ngx
 
-Read the section about [migration](https://paperless-ng.readthedocs.io/en/latest/setup.html#migration-to-paperless-ng) in the documentation. Its also entirely possible to go back to Paperless by reverting the database migrations.
+Read the section about [migration](https://paperless-ngx.readthedocs.io/en/latest/setup.html#migration-to-paperless-ng) in the documentation. Its also entirely possible to go back to Paperless by reverting the database migrations.
```
[the remark this answers is not in the document]
Comment (gador) [reply README.md:82]: yeah. the docs point at the master branch of our repo. All updates to the docs have been to the dev branch and are therefore not yet updated at readthedocs.

File: README.md [hunk 2]
```diff
@@ -98,15 +107,15 @@ If you want to implement something big: Please start a discussion about that! Ma
 
 Paperless has been around a while now, and people are starting to build stuff on top of it.  If you're one of those people, we can add your project to this list:
 
-* [Paperless App](https://github.com/bauerj/paperless_app): An Android/iOS app for Paperless. Updated to work with paperless-ng.
+* [Paperless App](https://github.com/bauerj/paperless_app): An Android/iOS app for Paperless. Updated to work with paperless-ngx.
```
[the remark this answers is not in the document]
Comment (bauerj) [reply README.md:116]: I can confirm it's compatible 😊

Maybe "...app for Paperless-ngx. Also works with the original Paperless and Paperless-ng."

**Comment by shamoon** [comment]
> Not really related to this PR, but I think the `README.md` should be restructured. The first wall of text focuses a lot on the difference between paperless-ng and the original paperless project.
> 
> Instead we could move the paperless-ngx explainer up higher, and move the Paperless compat stuff to a `## Original Paperless Compatibility/Migration` section?

Very much agree, it should be edited way down to give people a quick headline about the project, more screenshots, etc. Would you want that as part of this PR?

**Comment by qcasey** [comment]
Cool, I can draft that.

> Would you want that as part of this PR?

No, this PR's scope has already creeped a bit, I'd rather start fresh :)

**Review by bauerj — APPROVED** [review]

**Pull request merged**